src/tests/__init__.py

```diff
@@ -1,2 +1,5 @@
 import logging
-logging.disable(logging.CRITICAL)
+
+from buildercore.config import CONSOLE_HANDLER
+
+CONSOLE_HANDLER.setLevel(logging.CRITICAL)
```

Comment:
we still log to the file, which is useful to debug strange test failures

Reply:
> side effects don't belong into a __init__.py file I think

> BTW by default pytest will only show the logging or console output for failing tests. In that case I've usually done the opposite, set the default logging to DEBUG

Essentially there is too much output, so we disable most of the `CONSOLE_HANDLER` logs and leave `FILE_HANDLER` in place so that the logs can be seen in a file after the test run.

Yes, it could be, although maybe `src/tests/conftest.py` rather than the main one.

> BTW by default pytest will only show the logging or console output for failing tests. In that case I've usually done the opposite, set the default logging to DEBUG

> I saw pytest further up there, is that not what we are using?

Yes, we are exclusively running tests with `pytest` now

Reply:
> Essentially there is too much output, so we disable most of the `CONSOLE_HANDLER` logs and leave `FILE_HANDLER` in place so that the logs can be seen in a file after the test run.

When you say too much output, you mean too much shown or was it making it too slow? Because if it's failing, it seems good to have a lot of output. Usually it shouldn't fail and have no output. The file log however will have everything in it, for passing and failing tests.
If you see console output for passing tests, then there may be something wrong with the setup.

> Yes, it could be, although maybe `src/tests/conftest.py` rather than the main one.

By that you mean the default logging level DEBUG? Yes, for tests I meant (i.e. in conftest). When running the application the default logging level would usually be INFO.
Or do you mean another `conftest.py`?